framework/source/class/qx/ui/table/IColumnMenuItem.js

```diff
@@ -37,5 +29,27 @@ qx.Interface.define("qx.ui.table.IColumnMenuItem",
      * item is now visible.
      */
     changeVisible : "qx.event.type.Data"
+  },
+
+  members :
+  {
+    /**
+     * Set whether the table column associated with this menu item is visible
+     *
+     * @param value {Boolean} whether the table column associated with this
+     *   menu item is visible
+     */
+    setVisible : function (value) {
+      this.assertBoolean(value);
+    },
+
+
+    /**
+     * Get whether the table column associated with this menu item is visible
+     *
+     * @return {Boolean} whether the table column associated with this menu
+     *   item is visible
+     */
+    getVisible : function () {}
   }
```

Comment:
My opinion on this would still be, to completely remove the Interface, let Table directly use the `value` property (`getValue`, `setValue`, `"changeValue"` Event).
Yes it's an API-change, but a "save" one that can not be unnoticed for any application developer.

**Pro:** fixes this issue, less code
**Con:** Any application class, that implemented the interface will get an error (Interface is gone) and might have to use `value` then.

The "Conflicting method ..." warning for any implemented application class using a `visible` property will be thrown independent of this (which is correct!)